includes/class-wc-payments-account.php

```diff
@@ -51,6 +52,21 @@ public function clear_cache() {
 		delete_transient( self::ACCOUNT_TRANSIENT );
 	}
 
+	/**
+	 * Add clear account cache tool to WooCommerce debug tools.
+	 *
+	 * @param array $tools List of current available tools.
+	 */
+	public function debug_tool( $tools ) {
+		$tools['clear_wcpay_account_cache'] = [
+			'name'     => __( 'Clear WooCommerce Payments account cache', 'woocommerce-payments' ),
+			'button'   => __( 'Clear', 'woocommerce-payments' ),
+			'desc'     => __( 'This tool will empty the WooCommerce Payments account cache.', 'woocommerce-payments' ),
```

Comment:
What about altering the wording here: `This tool will reset the cached WooCommerce Payments account data.`. As non-native speaker, I would worry when read 'empty the... account cache' xD

Reply:
Yes, sounds bad read like this! 😄 What do you think about `This tool will clear the account cached values used in WooCommerce Payments.`?

Reply:
Sounds good =)